### PR TITLE
Feat/login component

### PR DESCRIPTION
**Pull request by alekhya66**

### CODE REVIEW

File: src/Components/fe-login/FeLogin.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  static get localizeNamespaces() {
+    return [
+      { 'fe-login': locale => import(`./translations/${locale}.js`) },
+      ...super.localizeNamespaces,
+    ];
+  }
+
+  static get properties() {
+    return {
+      username: { type: String },
```
Comment (Rajkumar-MP) [inline src/Components/fe-login/FeLogin.js]: Don't need this ur not passing any param

File: src/Components/fe-login/FeLogin.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      password: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    localize.locale = 'en-GB';
```
Comment (Rajkumar-MP) [inline src/Components/fe-login/FeLogin.js]: Dont set locale

**Review by Rajkumar-MP — CHANGES_REQUESTED** [review]

File: src/Components/fe-login/test/fe-login.test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import '../fe-login.js';
+
+describe('FeLogin', () => {
+  let element;
+  beforeEach(async () => {
+    element = await fixture(html`<fe-login></fe-login>`);
+  });
+
+  it('renders a username', () => {
+    const username = element.shadowRoot.querySelector('#username');
```
Comment (Rajkumar-MP) [inline src/Components/fe-login/test/fe-login.test.js:12]: this testcases will fail if we change the language as default

**Review by Rajkumar-MP — CHANGES_REQUESTED** [review]

File: src/Components/fe-login/FeLogin.js (hunk 1 above)
Comment (alekhya66) [reply src/Components/fe-login/FeLogin.js]: removed

File: src/Components/fe-login/FeLogin.js (hunk 2 above)
Comment (alekhya66) [reply src/Components/fe-login/FeLogin.js]: removed locale

File: src/Components/fe-login/FeLogin.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      }
+
+      h2 {
+        margin: 20px;
+      }
+    `;
+  }
+
+  inputValidation() {
+    const form = this.shadowRoot.querySelectorAll('lion-form');
```
Comment (Rajkumar-MP) [inline src/Components/fe-login/FeLogin.js]: is this working as expected, did u done check i storybook ?

File: src/Components/fe-login/FeLogin.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        font-size: 20px;
+      }
+
+      h2 {
+        margin: 20px;
+      }
+    `;
+  }
+
+  inputValidation() {
```
Comment (Rajkumar-MP) [inline src/Components/fe-login/FeLogin.js]: you didnt trigger the inputValiation function to do the submit

File: src/Components/fe-login/FeLogin.js [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+              new Required(null, {
+                getMessage: () => localize.msg('fe-login:passworderror'),
+              }),
+              new MinLength(8, {
+                getMessage: () => localize.msg('fe-login:default'),
+              }),
+            ]}"
+          ></lion-input>
+          <br />
+          <fe-footer .primary="${localize.msg('fe-login:submit')}"></fe-footer>
```
Comment (Rajkumar-MP) [inline src/Components/fe-login/FeLogin.js]: you passed label but not catched the event for submit trigger

**Review by Rajkumar-MP — CHANGES_REQUESTED** [review]

File: src/Components/fe-login/FeLogin.js [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+      firstFormElWithError.focus();
+      firstFormElWithError.classList.add('error-handle');
+
+      return;
+    }
+    this.dispatchEvent(
+      new CustomEvent('input-validation', { detail: serializedValue })
+    );
+
+    const loginData = ev.target.serializedValue;
```
Comment (Rajkumar-MP) [inline src/Components/fe-login/FeLogin.js]: this code is not needed, you already dispatched the event

**Review by Rajkumar-MP — APPROVED** [review]

**Review by Nayana-2607 — APPROVED** [review]

**Review by shadushazzz — APPROVED** [review]

**Pull request merged**